packages/deployer/webpack.config.js

```diff
@@ -16,4 +17,12 @@ module.exports = {
     mainFields: ['main'],
   },
   target: 'node',
+  plugins: [
+    new CopyPlugin({
+      patterns: [
+        { from: '../validator/dist/templates', to: 'templates' },
```

Comment:
Why don't w have to do this for the JS files copied using the `copy:webpack`?

Reply:
There's just one JS file and that's the `index.js`, the result of the Webpack run. Or maybe I don't understand the question.

Reply:
Ok, I'll try to break it down.

When you run `yarn build` in the `deployer` directory these targets are called:
* `compile` - TS transpilation outputting into the `dist` directory.
* `webpack` - Starting from `dist/src/handlers/aws/index.js` bundles everything that is imported/required into `.webpack`. This is the code for Lambdas. Now, with the validator integration, in order for this bundle to run we have to include validator templates. Since they are not imported in the validator code they need to be included separately. That's where the plugin comes in. It copies the template and convertor files needed for validation next to the bundle (in `.webpack`).
* `copy:webpack` - Copies the `.webpack` directory into `dist` directory so that when packaged node's lambda function is also included.
* `copy:terraform` - Copies the Terraform recipes into `dist` directory so that when packaged the deployment can be run from them.

Does it make sense now?

Reply:
Thanks, I was missing the part that the validator files are not required in the code and that's why they are left out of by webpack...

Maybe add link to your reply above the plugin (for future q)